indico/modules/events/editing/blueprint.py

```diff
@@ -62,12 +62,17 @@
 _bp.add_url_rule('/editing/api/service/disconnect', 'api_service_disconnect', service.RHDisconnectService,
                  methods=('POST',))
 _bp.add_url_rule('/editing/api/service/status', 'api_service_status', service.RHServiceStatus)
+_bp.add_url_rule('/editing/api/editables/files.zip', 'download_archive', management.RHDownloadArchive)
```

Comment:
I would not use a `.../api/...` URL for this one since it's something user-facing

Reply:
I wasn't actually very sure about this. I didn't like the idea of having it as part of the `/api` but in the end it is the same action as any other exposed by the API

Reply:
is it? Other APIs are called in the background, but for this one we need to redirect the user to it.

Actually, maybe the best option would be to do it like when printing badges etc, i.e. you make an API call that stores the (properly validated) data in redis (via GenericCache) and then returns a UUID to use in a URL to redirect to which starts the download.

Otherwise you'd need to create a HTML form to be able to submit everything which isn't amazing...